development-vm/replication/common-args.sh

```diff
@@ -42,7 +42,7 @@ SSH_CONFIG="../ssh_config"
 RENAME_DATABASES=true
 DRY_RUN=false
 # By default, ignore large databases which are not useful when replicated.
-IGNORE="event_store transition backdrop support_contacts"
+IGNORE="event_store transition backdrop support_contacts draft_content_store imminence"
```

Comment:
Adding these DBs seems like a good idea. 👍Can you still restore them if you need to, though? Have we tested them?

Reply:
There is an option to overwrite the IGNORE list.

Reply:
Have we tested that these can be replicated if we do override the list?